packages/site/src/lib/components/modals/DialectModal.svelte

```diff
@@ -0,0 +1,105 @@
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte';
+  import { t } from 'svelte-i18n';
+  import Modal from 'svelte-pieces/ui/Modal.svelte';
+  import Button from 'svelte-pieces/ui/Button.svelte';
+  import { dictionary } from '$lib/stores';
+
+  export let value: string;
+  interface IAlgoliaFacetsQuery {
+    facetHits: {
+      value: string;
+    }[];
+    exhaustiveFacetsCount: boolean;
+    processingTimeMS: number;
+  }
+
+  async function fetchDialects(): Promise<IAlgoliaFacetsQuery> {
+    const response = await fetch(
+      'https://XCVBAYSYXD.algolia.net/1/indexes/entries_prod/facets/di/query',
+      {
+        method: 'POST',
+        headers: {
+          'X-Algolia-Application-Id': 'XCVBAYSYXD', // App ID
+          'X-Algolia-API-Key': 'e6d98efb32d3dc2435dce7b97ea87c3e', // Public API key
+        },
+        body: JSON.stringify({
+          facetFilters: [[`dictId:${$dictionary.id}`]],
+          maxFacetHits: 100, // Algolia max possible https://www.algolia.com/doc/api-reference/api-parameters/maxFacetHits/
```

Comment:
If docs are obscure and you think might be useful in the future, always feel free to add a link. In this case, we usually use the Instantsearch docs, so I added a link to the rest api docs because this is our first use of them.